backend/game/battle/battle_logger.py

```diff
@@ -38,4 +38,4 @@ def add_to_event_list(self) -> None:
     def get_round_logs(self) -> str:
         packed_json = json.dumps(self.eventList)
         logging.info("Battle logs: " + packed_json)
-        return packed_json
+        return self.eventList
```

Comment:
Czy tutaj nie będzie problemu, iż nie jest czyszczony eventList? Czy zostało zmienione flow i nie ma z tym problemu? (co turę zamiast wysyłać kolejne ruchy, to wysyła coraz większe listy z także i poprzednimi ruchami)
+ wydaje mi się, że można wyrzucić linie dotyczące pakowania w json, jak nie ma takiej potrzeby

Reply:
Pakowanie jest teraz tylko dla logowania, w sumie można wyrzucić i robić str(eventList) ale nie wiem czy to duża różnica - choć zawsze jeden import json mniej :D 
 
Zostało zmienione flow: 
Teraz logi nie są po turze, tylko po rundzie. Znacznie ładniej to wychodzi w kodzie, a i tak chcemy mieć wszystkie te informacje. Front pewnie będzie "animować" to ruch po ruchu więc raczej nie robi mu różnicy tura a przy okazji ukrywamy że mamy tury a nie realtime :laughing:

Reply:
Można zrobić f-stringa -> (f"Battle logs: {self.eventList}"), wylatuje nam import i niepotrzebne ostatecznie dependency (chociaż nie żeby to jakoś mocno wpływało na całość).
Co do reszty to ok, seems legit 😄 
Zrobisz tę zmianę z jsonem, czy merge'ujemy i w kolejnym commicie którymś się to zmieni?